test/types.js

```diff
@@ -298,11 +298,11 @@ defineTypeTest('datetime_fraction', [
 ], '5.6.4');
 
 defineTypeTest('timestamp_fractional', [
-  'TIMESTAMP(3) NULL',
+  'TIMESTAMP(3) NULL'
 ], [
   ['"1970-01-01 00:00:01.123"'],
   ['"2038-01-18 03:14:07.900"'],
-  ['"2014-12-27 01:07:08.001"'],
```

Comment:
I'd love to have the comma, which is at the last element of an array, or an object (a hash). In practice, it's easier for you to cut & paste, or swap the line up or down (in Sublime Text, the shortcut is Command + Ctrl + :arrow_up: / :arrow_down: ) , even it's the last element. I think we kept a uniform style in this project.